### DISCUSSION TITLE
moonshinebrigade.com made with PicoCSS

### DISCUSSION THREAD

**Discussion by PaulHendriks** [picocss/pico#594] [Show and tell]
When i wanted to rewrite our website i was looking for something lightweight and actively maintained.
I've tried out several CSS frameworks, PicoCSS checked most boxes and is easy to use.
I use Sass to compile the stylesheets, maybe i'll dig into params and mixins some day. Today is not that day ;-)

Anyway, here's our new website, made with the pumpkin theme:

https://www.moonshinebrigade.com/

I've used the dialog element as a light/shadow box replacement.
Wavesurfer for the cool audio widget https://wavesurfer.xyz/

**Comment by cboecking** [comment]
Beautiful - thank you for sharing!
